### PR TITLE
added cypress tests for articles

### PR DESCRIPTION
**Pull request by mbilalhameed**

### CODE REVIEW

File: cypress_tests/cypress/integration/articles/articles_comments.spec.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+describe("Tests for comment section in articles", () => {
+    const url = "/en/sections/covid-19/odd-story/"
+
+    it("Test article comments when user is logged out and comments are open", () => {
+        cy.visitUrl("/admin/login/")
```
Comment (saqlainrasheed) [inline cypress_tests/cypress/integration/articles/articles_comments.spec.js:5]: use in variable

**Review by saqlainrasheed — CHANGES_REQUESTED** [review]

**Comment by geoo89** [comment]
When I run `make cypress`, it seems like there are some missing dependencies. Do they need to be specified somewhere?

```
django_1  | Traceback (most recent call last):
django_1  |   File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 330, in run_from_argv
django_1  |     self.execute(*args, **cmd_options)
django_1  |   File "/usr/local/lib/python3.8/site-packages/django/core/management/commands/runserver.py", line 61, in execute
django_1  |     super().execute(*args, **options)
django_1  |   File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 371, in execute
django_1  |     output = self.handle(*args, **options)
django_1  |   File "/usr/local/lib/python3.8/site-packages/django/core/management/commands/runserver.py", line 68, in handle
django_1  |     if not settings.DEBUG and not settings.ALLOWED_HOSTS:
django_1  |   File "/usr/local/lib/python3.8/site-packages/django/conf/__init__.py", line 82, in __getattr__
django_1  |     self._setup(name)
django_1  |   File "/usr/local/lib/python3.8/site-packages/django/conf/__init__.py", line 69, in _setup
django_1  |     self._wrapped = Settings(settings_module)
django_1  |   File "/usr/local/lib/python3.8/site-packages/django/conf/__init__.py", line 170, in __init__
django_1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
django_1  |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
django_1  |     return _bootstrap._gcd_import(name[level:], package, level)
django_1  |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
django_1  |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
django_1  |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
django_1  |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
django_1  |   File "<frozen importlib._bootstrap_external>", line 783, in exec_module
django_1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
django_1  |   File "/app/iogt/settings/cypress.py", line 1, in <module>
django_1  |     from .base import *
django_1  |   File "/app/iogt/settings/base.py", line 415, in <module>
django_1  |     from iogt.patch import *
django_1  |   File "/app/iogt/patch.py", line 1, in <module>
django_1  |     from translation_manager import utils
django_1  | ModuleNotFoundError: No module named 'translation_manager'
```

**Comment by geoo89** [comment]
Thanks! Looks good to me! It seems like the tests assume that some images are present, however they are not included in this PR and thus tests checking for the existence of images fail, e.g. [here](https://github.com/unicef/iogt/pull/791/files#diff-668dc3c4bb941706770f703d07c86925462b80c02999abf413d2b5363c68d5bbR26). Could you have a look and fix this?

For the SVG_to_PNG maps, you could also leave out those images and instead clear the svg_to_png map (django-admin -> svg to png maps -> delete all entries) so that they get created during the tests.

**Comment by sheralim012** [comment]
@geoo89 fixed tests cases, and added one image, and one svg icon to be used for all cases and added both to repository

**Pull request merged**